src/package1/packageDisplay.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Connection } from '@salesforce/core';
+
+export type Package1Display = {
+  MetadataPackageVersionId: string;
+  MetadataPackageId: string;
+  Name: string;
+  Version: string;
+  ReleaseState: string;
+  BuildNumber: number;
+};
+
+/**
+ * Executes server-side logic for the package1:display command
+ *
+ * @param connection
+ * @param id: MetadataPackageVersion
+ */
+export async function package1Display(connection: Connection, id: string): Promise<Package1Display[]> {
+  const query = `SELECT Id,MetadataPackageId,Name,ReleaseState,MajorVersion,MinorVersion,PatchVersion,BuildNumber FROM MetadataPackageVersion WHERE id = '${id}'`;
```

Comment:
@WillieRuemmele Could you create `MetadataPackageVersion` type and expose it in /src/interfaces/packagingSobject.ts.